AdyenCardScannerTests/AdyenCardScannerTests.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@testable import AdyenCardScanner
+import Testing
+
+struct AdyenCardScannerTests {
+
+    @Test func example() async throws {
+        // Write your test here and use APIs like `#expect(...)` to check expected conditions.
```

Comment:
Would be nice to have some tests with locally stored "card" image 🤔

Reply:
Yes, tests will be added in following PRs